features/step_definitions/checkout_screen_test.rb

```diff
@@ -0,0 +1,38 @@
+Given(/^I am on the index page$/) do
+  visit '/'
+end
+
+Then(/^I should be on the index page$/) do
+  expect(page).to have_current_path('/')
+end
+
+When(/^I visit the site$/) do
+  visit '/'
+end
+
+Given(/^the following dishes exists$/) do |table|
+  steps %q{
+    Given I have a dish called "Meatballs" with price "8.99"
+    And I have a dish called "Salad" with price "4.99"
+  }
+end
+
+Given(/^that I don't see "([^"]*)"$/) do |message_addded|
+  expect(page).not_to have_content message_addded
+end
+
+Given(/^I have no dishes in my basket$/) do
+  expect(@current_order).to be nil
+end
```

Comment:
Give this post a read and consider what I say about "Given" steps. https://medium.com/craft-academy/whats-so-good-about-cucumber-904c34d29f7b#.symptawut